### PR TITLE
debian: packaging fixes for jewel

### PR DESCRIPTION
**Pull request by ktdreyer**
Fixes the Trusty build failure in Gitbuilder.

### CODE REVIEW

**Comment by ktdreyer** [comment]
CC'ing @smithfarm @javacruft @alimaredia in case they want to review.

File: debian/rules [hunk 1]
```diff
@@ -67,7 +67,7 @@ clean:
 	  ltmain.sh missing
 	rm -f configure Makefile.in man/Makefile.in src/Makefile.in
 	rm -f src/acconfig.h.in
-	rm -f debian/ceph-base.init debian/radosgw.init debian/ceph-base/etc/logrotate.d/ceph.logrotate debian/radosgw.logrotate
+	rm -f debian/ceph-base.init debian/radosgw.init debian/ceph.logrotate debian/radosgw.logrotate
```
Comment (alimaredia) [inline debian/rules]: this change and the change above it undo the changes I made before merged, does ceph.logrotate not belong in debian/ceph-base/etc/logrotate.d/ anymore? Or did the changes I made not accomplish that? I remember it was a little tricky getting the logrotate files where I wanted them when I was making changes on them in November

Comment (ktdreyer) [inline debian/rules]: Yeah, I guess we'd gotten it wrong in the earlier commit (0cbe3dea69604730bc7dba7bc3844294f4c9ac5a) - sorry for leading you astray!

**Comment by smithfarm** [comment]
My Debian packaging skills are quite rusty. With that caveat:

`Reviewed-by: Nathan Cutler <ncutler@suse.com>`

**Comment by ktdreyer** [comment]
Re-pushed with packaging the libceph compressor files in the ceph-common .deb instead of the ceph-base .rpm (from Sage, "Re: packaging compressor plugins" thread on ceph-devel today.)

**Comment by tchaikov** [comment]
@alimaredia and @ktdreyer i've some nit at 0cbe3dea69604730bc7dba7bc3844294f4c9ac5a if they make sense to you guys, can we also address them in this pull request?

**Comment by ktdreyer** [comment]
Thanks Kefu, I've added the fix for ceph-base's dependencies.

**Comment by ktdreyer** [comment]
Re-pushed this PR without the compressor plugins. We'll have to handle those some other way. They need to be available to every package (librados2, librbd1, libcephfs1, plus all the daemons).
